CONTRIBUTING.md

```diff
@@ -38,7 +38,7 @@ To add a new rule, we suggest following these steps:
 8. If useful, create a folder named "original_files" and add relevant original files to that directory.
 9. Add the rule to the vale/styles/jargonLint or vale/styles/optional directory.
     - To jargonLint, if the rule is generally applicable.
```

Comment:
```suggestion
     - To `jargonLint` directory, if the rule is generally applicable.
```

Reply:
The line above states directories so I don't think this is necessary.